app/models/host.rb

```diff
@@ -59,6 +59,17 @@ def readonly?
     true
   end
 
+  def self.os_version_query(path, values, include = true)
```

Comment:
Observation: This fn might be in host_searching, if rubocop agrees.
Nit: `include` in function signature but `equal` as variable name outside of it.

Reply:
It doesn't agree :disappointed: